Pear/Controller/MessagesViewController.swift

```diff
@@ -18,7 +18,7 @@ class MessagesViewController: UIViewController {
 
     // MARK: - Private Data Vars
     private let databaseRef = Database.database().reference()
-    private var matchedUsers: [MatchedUser] = []
+    private var matchedUsers: [CommunityUser] = []
     private var matches: [TempMatchV2] = []
```

Comment:
talk to mat + noah about renaming all models with a V2? for example, `UserV2` should be `User` and `TempMatchV2` should be `TempMatch`